### ISSUE TITLE
Out of curiosity - Why is there a .kodi directory in /etc/skel?

### ISSUE THREAD

**Issue by MageJohn** [GalliumOS/galliumos-default-settings#5]
I noticed that there are default settings for kodi included in /etc/skel. I was wondering, why do they need to be there? This is just curiosity, though if there isn't any actual need I also wonder why it hasn't been removed.

**Comment by ghost** [comment]
Looking at the information for the commit in which it was included implies that it comes from the upstream package (xubuntu-default-settings), but when looking at the tree for that package I can't find a similar file. It's possible that it's been removed since then, but I'm honestly not too sure. I assume the default settings for Kodi are sane but I personally don't use it all too often so I've never paid it much mind.

Here's the commit I'm talking about, in case you're curious.
https://github.com/GalliumOS/galliumos-default-settings/commit/b5c9bc28b1eab849c969c22605430508603fcee6

**Comment by MageJohn** [comment]
You're right; I've got plenty of free time at the moment, so I took the few minutes to clone the xubuntu-default-settings git (https://git.launchpad.net/xubuntu-default-settings/) and look through the commit history, and I can't find any reference to etc/skel/.kodi, or anything similar. Could the file have ended up there by accident somehow?
